cmd-replay.c

```diff
@@ -166,64 +166,16 @@ static void print_field(struct ftrace_task_handle *task,
 		pr_out("| ");
 }
 
-static void setup_field(struct opts *opts)
+static void setup_default_field(struct opts *opts)
```

Comment:
I passed it as a function pointer because of the range handle routine as below:
```
static void setup_default_field(struct opts *opts)
{
        if (opts->range.start > 0 || opts->range.stop > 0) {
                if (opts->range.start_elapsed || opts->range.stop_elapsed)
                        add_field(&output_fields, field_table[REPLAY_F_ELAPSED]);
                else
                        add_field(&output_fields, field_table[REPLAY_F_TIMESTAMP]);
        }
        add_field(&output_fields, field_table[REPLAY_F_DURATION]);
        add_field(&output_fields, field_table[REPLAY_F_TID]);
}
```
Please let me know if there is a better way to handle this.

Reply:
I think the below looks a bit better:

```
static void setup_default_field(struct list_head *head, struct opts *opts)
{
   ...
}
```

Reply:
Ok. Done it.